FirebaseStorageSwift/Tests/Unit/StorageComponentTests.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseCore
+@testable import FirebaseStorage
+
+import FirebaseAppCheckInterop
+import FirebaseAuthInterop
+import SharedTestUtilities
+
+import XCTest
+
+class StorageComponentTests: XCTestCase {
+  static var app: FirebaseApp?
```

Comment:
Since we're configuring this in `setUp`, maybe we force unwrap here and avoid all the force unwraps in the tests?

Reply:
Does `static var app: FirebaseApp!`? or `static var app: FirebaseApp! = nil` work, or does it require a `let`?